blocks/core/ff_module/ff_module-profile-response-button/_ff_module-profile-response-button-renderer.js

```diff
@@ -10,9 +10,11 @@ module.exports = function() {
                 onSelect: function() {
                     console.log("onSelect");
                 },
-                isSelected: false,
+                uiState: "",
                 guid: "u42",
                 label: "Sally Student",
+                status: "Resubmission Requested",
+                mark: "B, 76%",
```

Comment:
I'm not sure: I haven't seen anywhere that allows you to pick between a mark and percentage. We let the user pick in the markbook though. Perhaps one to ask Viv, I'll ping her on Trello. For now, let's just ignore percentages.